code/game/objects/items/storage/toolbox.dm

```diff
@@ -46,13 +46,17 @@
 	new /obj/item/crowbar/red(src)
 	new /obj/item/weldingtool/mini(src)
 	new /obj/item/extinguisher/mini(src)
-	switch(rand(1,3))
+	switch(rand(1,5))
 		if(1)
 			new /obj/item/flashlight(src)
 		if(2)
 			new /obj/item/flashlight/glowstick(src)
 		if(3)
 			new /obj/item/flashlight/flare(src)
+		if(4)
+			new /obj/item/flashlight/flare(src)
```

Comment:
Is this intentional? You have the /flare item twice.

Reply:
Yes I wanted to have 2 chances for flare and 1 chance for signal flare.